internal/util/initcore/init_core.go

```diff
@@ -160,13 +161,27 @@ func InitRemoteChunkManager(params *paramtable.ComponentParam) error {
 	return HandleCStatus(&status, "InitRemoteChunkManagerSingleton failed")
 }
 
-func InitChunkCache(mmapDirPath string, readAheadPolicy string) error {
-	cMmapDirPath := C.CString(mmapDirPath)
-	defer C.free(unsafe.Pointer(cMmapDirPath))
-	cReadAheadPolicy := C.CString(readAheadPolicy)
-	defer C.free(unsafe.Pointer(cReadAheadPolicy))
-	status := C.InitChunkCacheSingleton(cMmapDirPath, cReadAheadPolicy)
-	return HandleCStatus(&status, "InitChunkCacheSingleton failed")
+func InitMmapManager(params *paramtable.ComponentParam) error {
+	mmapDirPath := params.QueryNodeCfg.MmapDirPath.GetValue()
+	if len(mmapDirPath) == 0 {
+		mmapDirPath = path.Join(paramtable.Get().LocalStorageCfg.Path.GetValue(), "mmap")
+	}
+	cMmapChunkManagerDir := C.CString(path.Join(mmapDirPath, "/mmap_chunk_manager/"))
+	cCacheReadAheadPolicy := C.CString(params.QueryNodeCfg.ReadAheadPolicy.GetValue())
+	defer C.free(unsafe.Pointer(cMmapChunkManagerDir))
+	defer C.free(unsafe.Pointer(cCacheReadAheadPolicy))
+	diskCapacity := params.QueryNodeCfg.DiskCapacityLimit.GetAsUint64()
+	diskLimit := uint64(float64(params.QueryNodeCfg.MaxMmapDiskPercentageForMmapManager.GetAsUint64()*diskCapacity) * 0.01)
```

Comment:
what does this 0.01 mean?

Reply:
milvus.yaml use precentage to represent this number.